documentation/run.DSpace7Entities.md

```diff
@@ -36,6 +36,25 @@ It will take approximately 5 minutes for the services to start completely.  Once
 - [http://localhost:8080/spring-rest](http://localhost:8080/spring-rest)
 - [http://localhost:3000](http://localhost:3000)
 
+### Ingest the Entities Test Dataset and Start DSpace 7 (Server Setup)
```

Comment:
Can we better describe the differences between "Server Setup" and "Localhost setup"?  I'm not sure this will be clear to all users.  My guess here is that by Server Setup, you are talking about a *remote/external* deployment of Docker on an external service (like AWS or similar).   Whereas "Localhost" is running it locally on your current machine.  I think we should describe the use cases for each, because non-developers may not know which one to choose.

Maybe they are better called "Local deployment" and "External deployment"?  With a brief paragraph describing when you'd use each one?